docs/using/recipes.md

```diff
@@ -549,3 +549,20 @@ RUN echo "from pyspark.sql import SparkSession" > /tmp/init-delta.py && \
     python /tmp/init-delta.py && \
     rm /tmp/init-delta.py
 ```
+
+## Add Custom Font in Scipy notebook
+
+The example below is a Dockerfile to load Source Han Sans with normal weight which is usually used for web.
+
+```dockerfile
+FROM jupyter/scipy-notebook:latest
+
+RUN PYV=$(ls "${CONDA_DIR}/lib" | grep ^python) && \
```

Comment:
This is a bit dangerous because some package might be called python-something, for example, but let's keep it this way - if someone faces a problem then, the fix will be easy enough.